stacks_queues/queue.py

```diff
@@ -12,50 +22,76 @@ class Queue:
     def __init__(self):
         self.store = [None] * INITIAL_QUEUE_SIZE
         self.buffer_size = INITIAL_QUEUE_SIZE
-        self.front = -1
+        self.front = 0
         self.rear = -1
         self.size = 0
-      
-
+    
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
         """
-        pass
+        if self.size == self.buffer_size:
+            raise QueueFullException('This will break it')
```

Comment:
Small suggestion: When raising an exception, it's helpful to give a description of _why_ the code is breaking
```suggestion
            raise QueueFullException('Queue full')
```